extensions/shared/register-jetpack-block.js

```diff
@@ -11,6 +12,13 @@ import getJetpackExtensionAvailability from './get-jetpack-extension-availabilit
 
 const betaExtensions = extensionList.beta || [];
 
+function requiresPlan( unavailableReason ) {
+	if ( endsWith( unavailableReason, '_plan_required' ) ) {
+		return unavailableReason.substring( 0, unavailableReason.length - '_plan_required'.length );
```

Comment:
`.endsWith( '_plan_required' )` would be like made for this but could be it's not polyfilled (no IE support)?

Reply:
Obsolete per 87d15d8